internal/cli/users.go

```diff
@@ -76,17 +77,18 @@ var (
 		Help:      "Number of users, that match the search criteria, to retrieve. Minimum 1, maximum 1000. If limit is hit, refine the search query.",
 	}
 	userImportTemplate = Flag{
-		Name:       "Template",
-		LongForm:   "template",
-		ShortForm:  "t",
-		Help:       "Name of JSON example to be used.",
+		Name:      "Template",
+		LongForm:  "template",
+		ShortForm: "t",
+		Help: "Name of JSON example to be used. Cannot be used if the '--users-body' flag is passed. " +
+			"Options include: 'Empty', 'Basic Example', 'Custom Password Hash Example' and 'MFA Factors Example'.",
 		IsRequired: false,
 	}
-	userImportTemplateBody = Flag{
-		Name:       "Template Body",
-		LongForm:   "template-body",
+	userImportBody = Flag{
+		Name:       "Users Body",
+		LongForm:   "users-body",
```

Comment:
I think that we should prefer shorter flag names wherever possible, for usability reasons. Also I suspect changing this pattern to "resources-property" would cause a bigger inconsistency, requiring the renaming of a lot more flags.

`users` is fine for this, I think.

Reply:
Renamed it to users in [`c092246` (#735)](https://github.com/auth0/auth0-cli/pull/735/commits/c0922467daf50c03eba3f3f10487f2e6e2a98759). 

But I also removed the short form as it was conflicting and replacing with with anything else than `u` doesn't really work so well.

Reply:
Consider having `-u` be the short form for `users` and `-U` the short version for `upsert`.

Reply:
Good point 👍🏻 [`de8a9a4` (#735)](https://github.com/auth0/auth0-cli/pull/735/commits/de8a9a4cbddbedb200edaea3b66565e9e94a192d)

For some reason I thought short forms had to always be lower case.